addon/src/motions/adjust-color.ts

```diff
@@ -1,20 +1,35 @@
 import { rAF } from '../-private/concurrency-helpers';
-import Motion from '../-private/motion';
+import Motion, { BaseOptions } from '../-private/motion';
```

Comment:
@wagenet should we do here `import Motion, { type BaseOptions } from '../-private/motion';`?
The addon uses `"typescript": "^4.6.2"` at build time so we can safely use type modifiers on import